ui/app/templates/components/database-connection.hbs

```diff
@@ -272,7 +272,7 @@
     </div>
 
     {{! Statements Edit Section }}
-    {{#unless (and @model.plugin_name (not @model.statementFields))}}
+    {{#if (not (and @model.plugin_name (not @model.statementFields)))}}
```

Comment:
in general, is this how you're approaching removing nested unless? (e.g. using if not)

Reply:
PS - i hate the word unless. it hurts my brain to think through it, which I know it shouldn't, but still does.

Reply:
I'd say it's a case by case basis and there is actually a different lint rule that says not to use if not for a simple condition but in this case it was the easy way to refactor.

With this rule now set to error it will be easier for the developer to rethink the condition at the time. It's a bit tricky going in after the fact and trying to rework it.

Reply:
> PS - i hate the word unless. it hurts my brain to think through it, which I know it shouldn't, but still does.

:grin: it seems like they are really trying to move people away from it so that's good! Basically they only want you using it for a simple condition like `{{unless this.foo}}`